### ISSUE TITLE
Menu Items Stop Working When Frontend Page Type set to Flex Object

### ISSUE THREAD

**Issue by thekenshow** [trilbymedia/grav-plugin-flex-objects#46]
I have two different sites under development, one with the RT Vermilion theme and another using Helium, and on both sites the menus stop working when Frontend Page Type is set to Flex. 

The HTML for the menu changes from something this:

```
<li class="g-menu-item g-menu-item-features-particles g-menu-item-type-link">
	<a class="g-menu-item-container" href="/features/particles">
		<span class="g-menu-item-content">
			<span class="g-menu-item-title">
				Particles
			</span>
		</span>
	</a>
</li>
```

To something like this (the `<a>` tag disappears):

```
<li class="g-menu-item g-menu-item-features-particles g-menu-item-type-separator">
	<div class="g-menu-item-container" data-g-menuparent="">
		<span class="g-separator g-menu-item-content">
			<span class="g-menu-item-title">
				Particles
			</span>
		</span>
	</div>
</li>
```

I'm using Grav 1.7rc3, Gantry 5.4.32 (testing with Gantry CI build did not seem to make any difference), PHP 7.1.33.

**Comment by mahagr** [comment]
Yup, verified the bug.

**Comment by mahagr** [comment]
Should be fixed for Grav 1.7.0-rc.4. :)

**Comment by thekenshow** [comment]
Excellent, thanks!
